src/internals/TwoWayBinding.ts

```diff
@@ -174,7 +247,8 @@ function isInStrictMode(widget: WidgetInterface) {
 
 function bindingToString(localPath: LocalPath, targetPath: TargetPath): string {
   try {
-    return `"${localPath.join('.')}" <-> "${targetPath[1]}.${targetPath[2]}"`;
+    const arrow = targetPath[0] || '<->';
```

Comment:
Prefer `??` over `||` to prevent falsiness side-effects?

Reply:
No falsy value would be valid here.